### DISCUSSION TITLE
Balena OS and reComputer R1000

### DISCUSSION THREAD

**Discussion by KasunThushara** [Seeed-Studio/OSHW-RPi-Series#24] [On Development]
We are currently adding functionalities to our reComputer R1000 to work with **BalenaOS**, allowing our users to run Docker containers on embedded devices and interact with **Balena Cloud**. We would like to hear your thoughts and are ready to help make this happen.

**Comment by ackPeng** [comment]
At present, R1000 already supports balenaOS. It can refer to the wiki (https://wiki.seeedstudio.com/reComputer_r1000_balena/) to use balena
